src/app/campaign-list/card/campaign-card.component.scss

```diff
@@ -1,18 +1,20 @@
+@import '~src/sass/variables';
+
 :host {
   display: block;
   background-color: white;
```

Comment:
should there be a $prx-white or some kind of grey?

Reply:
Switched over to the lightest gray, but we could define a PRX white (although I'd lean towards just using `black` or `white` if we're just going to define a variable that is `#000000` or `#ffffff`)